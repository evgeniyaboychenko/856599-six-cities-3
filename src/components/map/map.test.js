import React from 'react';
import renderer from 'react-test-renderer';
import Map from './map.jsx';
jest.mock(`./map`);

const OFFER_IMAGES = [`room.jpg`, `apartment-01.jpg`];
const OFFER_NAMES = [`Canal View Prinsengracht`, `Nice, cozy, warm big bed apartment`];
const OFFER_TYPES = [`apartment`, `privet room`];
const COORDINATES = [[52.3909553943508, 4.85309666406198], [52.369553943508, 4.85309666406198], [52.3909553943508, 4.929309666406198], [52.3809553943508, 4.939309666406198]];

const offerCards = [
  {
    id: `1`,
    coordinates: COORDINATES[0],
    photos: OFFER_IMAGES,
    name: OFFER_NAMES[0],
    rating: 2,
    price: 100,
    type: OFFER_TYPES[0],
    isPremium: true,
  },
  {
    id: `2`,
    coordinates: COORDINATES[1],
    photos: OFFER_IMAGES,
    name: OFFER_NAMES[1],
    rating: 3,
    price: 120,
    type: OFFER_TYPES[1],
    isPremium: false,
  }
];

it(`should Map render correctly`, () => {
  const tree = renderer.create(
      <Map
        offerCards = {offerCards}
      />, {
        createNodeMock: () => {
          return {};
        }
      }).toJSON();

  expect(tree).toMatchSnapshot();
});

