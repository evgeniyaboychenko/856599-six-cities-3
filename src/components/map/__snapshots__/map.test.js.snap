// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should Map render correctly 1`] = `null`;
